pkg/systemd/terminal.scss

```diff
@@ -5,7 +5,7 @@
 .console-ct-container {
   block-size: 100%;
   display: grid;
-  grid-template-rows: auto 1fr;
+  grid-template-rows: auto auto 1fr;
```

Comment:
I don't know enough about grids to see what that does. I figure it's about accomodating the alert, but what if the alert isn't rendered?